examples/classification/script.jl

```diff
@@ -0,0 +1,184 @@
+# # Stochastic Variational Classification
+#
+# This example demonstrates how to carry out non-conjugate Gaussian process
+# inference using the stochastic variational Gaussian process (SVGP) model. For
+# a basic introduction to the functionality of this library, please refer to the
+# [User Guide](@ref).
+#
+# ## Setup
+
+using SparseGPs
+using ParameterHandling
+using Zygote
+using Distributions
+using LinearAlgebra
+using Optim
+
+using Plots
+default(; legend=:outertopright, size=(700, 400))
+
+using Random
+Random.seed!(1234)
+#md nothing #hide
+
+# ## Generate some training data
+#
+# For our binary classification model, we will use the standard approach of a
+# latent GP with a Bernoulli likelihood. This results in a generative model that
+# we can use to produce some training data.
+#
+# First, we define the underlying latent GP
+# ```math
+# f \sim \mathcal{GP}(0, k(\cdot, \cdot'))
+# ```
+# and sample a function `f`.
+
+k_true = [30.0, 0.5]
```

Comment:
could you find a more interesting sample? either a different seed or perhaps somewhat shorter lengthscale...

Reply:
The reason for the simple sample is that optimisation gets a bit dodgy when it's more interesting :)

Reply:
fair enough!

Reply:
though I think that might be an issue with the parametrisation; normally, there shouldn't be any issues fitting 1D toy datasets!

Reply:
https://github.com/JuliaGaussianProcesses/SparseGPs.jl/issues/35